### DISCUSSION TITLE
Civ Clone projects

### DISCUSSION THREAD

**Discussion by Blakeley00** [HonkiTonk/Civ-Klon#5] [General]
Hi there, youtube won't let me post the link on your stream so I'll have to use this instead! I'm the moderator of the Civ1 forum at CivFanatics and there's a big Civ1 clone project thread there covering others attempting what you are (they've got their own githubs too). I've just posted about your work there but you might want to check out theirs! :)
https://forums.civfanatics.com/threads/civone-an-open-source-remake-of-civilization-1.535036/

If it's of interest to you some of the same guys along with others are also working on a Civ2 clone project in this thread:
https://forums.civfanatics.com/threads/making-a-clone-of-civ-ii.649543/page-3#post-15983617

**Comment by HonkiTonk** [comment]
Suddenly people come here and want things from me, crazy. But I wrote something in your forum. However, I have to say that I currently have relatively little interest in other clones/games. I hope to sell my game one day, so I spend most of my time developing it.
